src/docker/dockerclient.h

```diff
@@ -0,0 +1,24 @@
+#ifndef DOCKER_CLIENT_H
+#define DOCKER_CLIENT_H
+
+#include <json/json.h>
+#include <iostream>
+#include <string>
+#include "restclient.h"
+using namespace std;
+
+class DockerClient {
+ public:
+  DockerClient() : uri_("http://127.0.0.1") { RestClient::init(); }
+  DockerClient(string host) : uri_(host) { RestClient::init(); }
+  virtual ~DockerClient() { RestClient::disable(); }
+  string param(string param_name, string param_value);
```

Comment:
`param` methods don't look like the docker client class methods rather some URI class methods if it's needed at all.